src/ServiceControlInstaller.Engine/Validation/ConnectionStringValidator.cs

```diff
@@ -17,7 +18,7 @@ class ConnectionStringValidator
         public static void Validate(IServiceControlInstance instance)
         {
             var validator = new ConnectionStringValidator(instance.ConnectionString, instance.ServiceAccount);
-            if (instance.TransportPackage.ZipName.Equals("SQLServer", StringComparison.OrdinalIgnoreCase))
+            if (instance.TransportPackage.Name.Equals(TransportNames.SQLServer, StringComparison.OrdinalIgnoreCase))
```

Comment:
Maybe outside of the scope of this PR but I wonder if this shouldn't just go into the transport definitions. i.e. Find the associated transport definition and then ask it to validate the connection string.

Reply:
I propose to leave this out of scope.